_develop/connect/get-started.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: learn
+permalink: /:collection/:path.html
+---
+# Get Started
+{:.no_toc}
+
+Blockstack Connect is a Javascript library for integrating your application with Stacks v2. With Connect, you get some big benefits:
+
+<!-- -  -->
```

Comment:
@markmhx could we get the top benefits of Connect for devs listed here?

Reply:
@hstove and @aulneau may be best for articulating these from a developer perspective

Reply:
Although it seems we want the same content here as here: https://github.com/blockstack/docs.blockstack/pull/540#pullrequestreview-392416768